gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/fakes/spring/FakeConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package fakes.spring;
+
+import fakes.FakeAnalyticsQueryService;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class FakeConfiguration {
```

Comment:
Nice ! 💛
Maybe we should take time to share it with the dev team. Just to explain when, how and why we should use Fakes instead of mocks for example.